### PR TITLE
Gui testcases style update

### PR DESCRIPTION
**Pull request by maryvictol**
This PR 
- updates old testcases to common table style
- adds testcases description
- adds lists of testcases

### CODE REVIEW

File: docs/testcases/UI/Run_tools_in_sandbox/1825_allow_to_run_hosted_applications/1825_3.md [hunk 1]
```diff
@@ -1,52 +1,42 @@
 # [Manual] Check run "hosted" applications (several ports)
+
+Test verifies run "hosted" applications with several0 ports
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/1825_allow_to_run_hosted_applications/1825_3.md]: Fix to `... with several ports`

File: docs/testcases/UI/Pipeline_library/Pipeline_library.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+This folder contains testcases to check Pipeline library.
+
+| Case ID                                                                            | Description/name |
+|------------------------------------------------------------------------------------|---|
+| [**1353**](Pipeline_details/1353.md)                                               | Check pipeline sources for previous draft versions |
+| [**1517_1**](Pipeline_configuration/1517_custom_node_images_for_runs/1517_1.md)    | Check custom node image for the pipeline run |
+| [**1517_2**](Pipeline_configuration/1517_custom_node_images_for_runs/1517_2.md)    | Check node reuse after the custom node image run |
+| [**1506_1**](Folder_navigation/1506_pipelines_and_storages_repositories/1506_1.md) | Check "All pipelines" repository |
+| [**1506_1**](Folder_navigation/1506_pipelines_and_storages_repositories/1506_1.md) | Check "All storages" repository |
```
Comment (NShaforostov) [inline docs/testcases/UI/Pipeline_library/Pipeline_library.md]: Fix to `| [**1506_2**](Folder_navigation/1506_pipelines_and_storages_repositories/1506_2.md) | ...`

File: docs/testcases/UI/Sample_Processing_Module/1589.md [hunk 3]
```diff
@@ -14,39 +16,19 @@
...
+| 1 | Open the **Library** page | |
+| 2 | Open the folder created at step 3 of the preparations | |
+| 3 | Click the **Metadata** item | |
+| 4 | Click the "SampleSet <...>" item in the list | <li> the table contains the column "Created Date" <li> there are no empty cells in this column |
+| 5 | Click the "Metadata" item in the breadcrumbs-control at the top of the page | |
+| 6 | Click the "Sample <...>" item in the list | <li> the table contains the column "Created Date" <li> there are no empty cells in this column |
+| 7 | Click the "**+ Add instance**" button | |
+| 8 | Specify any valid "**Instance ID**" into the correspondding field | |
+| 9 | Click the **Create** button | <li> the table contains the row with ID specified at step 8 <li> there is a non-empty value of the "Created Date" in this row |
+| 10 | Save the "**Created Date**" value in the row with ID specified at step 8 | In the table: <br><li> there is a new column with the name equals to **Key** value specified at step 12 <li> there is an only single column "Created Date" <li> "**Created Date**" value in the row with ID specified at step 8 equals to the saved at step 10 |
```
Comment (NShaforostov) [inline docs/testcases/UI/Sample_Processing_Module/1589.md]: Fix expected result to: `... name equals to **Key** value specified at step 13...` and then move this whole expected result to the step 14 (_it was a mistake in the original case_)

File: docs/testcases/UI/Sample_Processing_Module/1589.md [hunk 4]
```diff
@@ -14,39 +16,19 @@
...
+| 11 | Click the row with ID specified at step 8 | |
+| 12 | In the **Attributes** panel, click the "**+ Add**" button | |
+| 13 | Specify any valid values for **Key** and **Value** fields (e.g. `key1`, `value1`) | |
+| 14 | Click the **Add** button | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Sample_Processing_Module/1589.md]: Move here the expected result from step 10

File: docs/testcases/UI/System_Management/2232-2444_NAT_Gateway/2232_4.md [hunk 5]
```diff
@@ -15,12 +15,12 @@ Test verifies that it isn't possible to add route to the existing route with res
 | 5 | Store *External IP* for the route added at case [2232_2](2232_2.md) | |
 | 6 | Click **Add Route** button | |
 | 7 | Input `<Server_name_1>` from case [2232_2](2232_2.md) into the ***Server name:*** field |  |
-| 8 | Mark ***Specify IP address*** checkbox | ***IP*** field with resolved IP address appears |
+| 8 | Mark ***Specify IP address*** checkbox | ***IP*** field with resolved IP address appears | |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/2232-2444_NAT_Gateway/2232_4.md]: Remove extra `|` symbol in the end of the row

File: docs/testcases/UI/System_Management/2232-2444_NAT_Gateway/2232_9.md [hunk 6]
```diff
@@ -22,9 +22,9 @@ Test verifies that
 | 5 | Click **Add Route** button | |
 | 6 | Input `<Server_name_3>` into the ***Server name:*** field |  |
 | 7 | Input `80` into the ***Port:*** field | |
-| 8 | Click **Add Port** button | **Add** button becomes disabled |
+| 8 | Click **Add Port** button | **Add** button becomes disabled | |
```
Comment (NShaforostov) [inline docs/testcases/UI/System_Management/2232-2444_NAT_Gateway/2232_9.md]: Remove extra `|` symbol in the end of the row

**Review by NShaforostov — APPROVED** [review]
Fix minor notes before the merge (see comments).

File: docs/testcases/UI/Run_tools_in_sandbox/1825_allow_to_run_hosted_applications/1825_3.md (hunk 1 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/1825_allow_to_run_hosted_applications/1825_3.md]: Done

File: docs/testcases/UI/Pipeline_library/Pipeline_library.md (hunk 2 above)
Comment (maryvictol) [reply docs/testcases/UI/Pipeline_library/Pipeline_library.md]: Done

File: docs/testcases/UI/Sample_Processing_Module/1589.md (hunk 3 above)
Comment (maryvictol) [reply docs/testcases/UI/Sample_Processing_Module/1589.md]: Done

File: docs/testcases/UI/Sample_Processing_Module/1589.md (hunk 4 above)
Comment (maryvictol) [reply docs/testcases/UI/Sample_Processing_Module/1589.md]: Done

File: docs/testcases/UI/System_Management/2232-2444_NAT_Gateway/2232_4.md (hunk 5 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/2232-2444_NAT_Gateway/2232_4.md]: Done

File: docs/testcases/UI/System_Management/2232-2444_NAT_Gateway/2232_9.md (hunk 6 above)
Comment (maryvictol) [reply docs/testcases/UI/System_Management/2232-2444_NAT_Gateway/2232_9.md]: Done

**Pull request merged**